### PR TITLE
Support multiple callbacks

### PR DESCRIPTION
**Pull request by czwartaoslpoj**

### CODE REVIEW

File: compredict/client.py [hunk 1]
```diff
@@ -80,8 +81,24 @@ def verify_peer(self, option: str):
...
 
+    def set_callback_urls(self, callback_url: Union[list, str]) -> str:
+        """
+        Accept list of urls and format them into one string with dividing '|' in between.
+        This is the format accepted by ai core.
+        :param callback_url: list of callback urls
+        :return: one callback_url string
+        """
+
+        if type(callback_url) == list:
```
Comment (uzi0espil) [inline compredict/client.py]: Here it is better to use isinstance(callback_url, list) than type(callback_url). Here is [why](https://switowski.com/blog/type-vs-isinstance) FYI.

File: compredict/client.py [hunk 2]
```diff
@@ -80,8 +81,24 @@ def verify_peer(self, option: str):
     def last_error(self) -> Error:
         return self.connection.last_error
 
+    def set_callback_urls(self, callback_url: Union[list, str]) -> str:
```
Comment (uzi0espil) [inline compredict/client.py]: Add `Union[List[str], str]` to be more precise where `List` is coming from `from typing import List`. Do the same for def run_algorithm.

Comment (uzi0espil) [reply compredict/client.py]: In addition, I think set_callback_urls will only be ran within the class, so we can make it protected, so I suggest to name the function `_set_callback_urls`

File: compredict/client.py [hunk 3]
```diff
@@ -192,10 +211,11 @@ def run_algorithm(self,
                       version: Optional[str] = None,
                       evaluate: bool = True,
                       encrypt: bool = False,
-                      callback_url: Optional[str] = None,
-                      callback_param: Optional[dict] = None,
+                      callback_url: Optional[Union[str, list]] = None,
+                      callback_param: Optional[Union[dict, list]] = None,
```
Comment (uzi0espil) [inline compredict/client.py]: same as callback_url, lets be more precise on the type of list in callback_param: `callback_param: Optional[Union[dict, List[dict]]]`

**Review by uzi0espil — CHANGES_REQUESTED** [review]
Looks great, few changes and we can close this issue :)

File: compredict/client.py (hunk 3 above)
Comment (czwartaoslpoj) [reply compredict/client.py]: Changed.

File: compredict/client.py (hunk 2 above)
Comment (czwartaoslpoj) [reply compredict/client.py]: Done.

File: compredict/client.py (hunk 1 above)
Comment (czwartaoslpoj) [reply compredict/client.py]: Cool. Didn't know about it. Done.

**Pull request merged**